robotd/devices.py

```diff
@@ -349,10 +359,14 @@ def _command(self, *args):
 
                 if line.startswith(b'@'):
                     returned_command_id_str, line = line[1:].split(b' ', 1)
-                    returned_command_id = int(returned_command_id_str.decode('utf-8')) & 0xffff
+                    returned_command_id = int(
+                        returned_command_id_str.decode('utf-8'),
+                    ) & 0xffff
 
                     if returned_command_id != command_id:
-                        print('Got response for different command, ignoring...')
+                        print(
```

Comment:
Oh, What fresh hell is this?! 😱

Reply:
How else would you wrap this? (This is the style we use at work; I agree it looks a bit odd here, but it works _really_ well for longer or more complicated examples)

Reply:
My main issue was the trailing comma on the arguments. 

I think my eyes are used to seeing Python aligned to 120 length over 80, which makes this break seem unnecessary, but that's just my training.

Reply:
Ah, I see. This is from `flake8-commas` which is handy as it ensures that when adding more things you don't get the diff noise from adding commas on the preceding lines.

I'm open to making the boundary a little longer (79 is the default), though I'm not sure I'd go as far as 120 characters. Maybe 90?

Reply:
I _really_ don't like trailing commas like that. They make sense on multi-element, multi-line cases, but in this case, it just looks wrong. Although, i'm fairly happy to be outvoted, assuming the linter enforces whatever standard we decide on.

79 for me is always a little too short. I think 90 works for me! On an unrelated note, would be good to enforce the same linter rules on all repos

Reply:
Yeah, if we didn't have a hard limit on line lengths I'd definitely leave this unwrapped.

3e16f97